webanno-ui-core/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/core/WicketApplicationBase.properties

```diff
@@ -66,13 +66,15 @@ filesToImport=Files to import
 readonly=Read-only
 required=Required
 visible=Visible
+featureShowHover=Show in feature text in mouse over 
```

Comment:
`... in popup`?

Reply:
Yes, thats probably easier to understand for the users. Changed in 232027fd0f4efd0a246443db1ba8b87571228d91